CHANGELOG.md

```diff
@@ -1,25 +1,31 @@
 # Cromwell Change Log
 
-## 29
+## 29 Release Notes
 
 ### Breaking Changes
 
-* Request timeouts for HTTP requests on the REST API now return a 503 status code instead of 500. The response for a request timeout is no longer in JSON format.
-* The metadata endpoint no longer returns gzipped responses by default. This now needs to be explicitly requested with an `Accept-Encoding: gzip` header 
+* **Command line**  
+In preparation for supporting CWL scripts (yes, you read that right!), we have extensively revised the Command Line in Cromwell 29. For more details about the usage changes, please see the
+[README](https://github.com/broadinstitute/cromwell#command-line-usage). And stay tuned to the [WDL/Cromwell blog](https://software.broadinstitute.org/wdl/blog) over the next couple of months for more news about CWL.
 
-* Command line usage has been extensively revised for Cromwell 29.  Please see the
-[README](https://github.com/broadinstitute/cromwell#command-line-usage) for details.
+* **Request timeouts**   
+HTTP requests to the REST API returns a 503 status code instead of 500 in accordance with Akka HTTP. The response for a request timeout will now be plain text, rather than a JSON format.
 
-* The engine endpoints are now served under `/engine`. Previousely engine endpoints were available under
-`/api/engine`. Workflow endpoints are still served under `/api/workflows`. The setting `api.routeUnwrapped` has been
-retired at the same time.
+* **Metadata endpoint**  
+The response from the metadata endpoint can be quite large depending on your workflow. To help reduce the file size Cromwell can gzip encode the metadata prior to sending it back. To enable gzipped responses, send the `Accept-Encoding: gzip` header. By default the metadata endpoint does not gzip encode responses.  
```

Comment:
"By not forcing it on you" you mean that users now have to opt-in to gzipping their metadata file, right? I don't want to say that Cromwell no longer encodes the files in order to behave how an API technically should.

How about:
You can now opt-in to have Cromwell gzip encode your metadata file, in order to reduce file size, by sending the header.